raiden/network/transport/matrix.py

```diff
@@ -1177,3 +1227,66 @@ def _get_room_ids_for_address(
             ]
 
         return room_ids
+
+    def _clean_unused_rooms(self, _address_to_room_ids: Dict[AddressHex, List[_RoomID]] = None):
```

Comment:
Maybe `_leave_unused_rooms` would make it clearer what the method actually does.

Reply:
Used clean because it also cleans the mappings on server, but both would fit